### PR TITLE
Client basic rewrite

### PR DESCRIPTION
**Pull request by NPM8**
Mostly is a small rewrite and changes in project structure to prepare a better development environment.

**The code in here needs to be tested before merged.**

Please add your critic to this project structure and if you want additional changes which should be done in this PR.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
+        "sourceType": "module",
+        "project": "./tsconfig.json"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+      "import/prefer-default-export": ["off"],
+      "class-methods-use-this": "off",
+      "import/extensions": "off"
```
Comment (dominik-korsa) [inline .eslintrc.json]: Fix indentation

File: src/client/client.ts [hunk 2]
```diff
@@ -82,4 +88,43 @@ export class Client {
     });
     return true;
   }
+
+  /**
+   * Parsing login respond to find `wresult` in hidden form in respond from UONET API.
```
Comment (doteq) [inline src/client/client.ts]: Zamieniłbym na "Parsing login respond to get last used symbols by user."

Comment (dominik-korsa) [reply src/client/client.ts]: Właściwie rzeczownik to będzie *response* (*odpowiedź*)

Comment (NPM8) [reply src/client/client.ts]: Yeah, that's a great idea.

**Comment by NPM8** [comment]
Those from `.idea` are auto-generated by JetBrains IDEA and I think we shouldn't change them.

File: src/client/client.ts (hunk 2 above)
Comment (NPM8) [reply src/client/client.ts]: Done

File: .eslintrc.json (hunk 1 above)
Comment (NPM8) [reply .eslintrc.json]: Done

**Pull request merged**